### DISCUSSION TITLE
Details side panel Eject Block

### DISCUSSION THREAD

**Discussion by Bitello** [Budibase/budibase#9719] [Help] [answered]
Hello!

I tried to eject a block from a "Details side panel" and... it didn't eject completely (it worked even before one of the last updates).
"New row side panel" is working correctly.

![image](https://user-images.githubusercontent.com/22456825/219434592-d3e7e7a1-caf1-42a3-a92c-9ccf0347ab16.png)

I use version 2.3.16
Is it just me this is happening to?

**Comment by melohagan** [comment, marked as answer]
Hey @Bitello 

Are you able to provide reproduction steps? 
Was this an internal or external DB? 
Did you see any error messages (including in the Console tab?)

**Reply by Bitello** [reply]
Hi @melohagan

Before opening this discussion 3 times, I autogenerated the same page and ejected the blocks... always with the same result... I exited the application... came back... logged out... logged in again... and with the same problem... (I even made a printscreen)

I went to do the step by step to present the error and .... it worked ...

Anyway...

Sorry... and thank you
